site/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -176,12 +176,17 @@ exception expires in Chrome 96, and the exception only applies to Desktop
 Chrome.
 
 1. [Request a token]({{origin_trial.url}}) for your origin.
-2. Add the token to your pages. There are two ways to do that:
-   - Add a `<meta>` tag to the head of each page. For example,
-     this may look something like: `<meta http-equiv="origin-trial" content="TOKEN_GOES_HERE">`
-   - If you can configure your server, you can also add the token using an
-     `Origin-Trial` HTTP header. The resulting response header should look
-     something like: `Origin-Trial: TOKEN_GOES_HERE`
+2. Add the token to your pages using an `Origin-Trial` HTTP header. The
+   resulting response header should look something like: `Origin-Trial:
+   TOKEN_GOES_HERE`
+
+{% Aside 'caution' %}
+You can only enable the origin trial using an `Origin-Trial` HTTP header. Please
+do not use a meta tag.
```

Comment:
```suggestion
You can only enable the origin trial using an `Origin-Trial` HTTP header.
Do not use a meta tag for this purpose.
```

Reply:
Just a note that it's best practice to be direct and avoid using "please" for instructions, it's not impolite :)